packages/module/patternfly-docs/content/extensions/virtual-assistant/examples/VirtualAssistant/AssistantMessageWithFollowup.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import VirtualAssistant from '@patternfly/virtual-assistant/dist/dynamic/VirtualAssistant';
+import AssistantMessageEntry from '@patternfly/virtual-assistant/dist/dynamic/AssistantMessageEntry';
+import { From } from '@patternfly/virtual-assistant/dist/dynamic/types';
+
+export const AssistantMessageWithFollowup: React.FunctionComponent = () => (
+  <VirtualAssistant>
+    <AssistantMessageEntry
+      message={ { content: "Here are a few things I can help you with. Select an option below or type in your questions:", options: [ { title: "Change my personal information", payload: "This is Payload" }, { title: "Show my critical vulnerabilities", payload: "This is Payload 2" }, { title: "Get recommendations from Advisor", payload: "This is Payload 3" } ], messageId: "1", from: From.ASSISTANT, isLoading: false } }
```

Comment:
Hi! 

I know that this is still draft, but trying to give feedback early!

I don't think we should handle stuff like `messageId` in here (unless that's strictly related to react's display stuff!)
Also `blockInput`, `preview` and `ask` sound like a application specific sort of things.

We should keep this simple as building blocks and let the application (not here) put those blocks together, I would only add the message components directly

```
<MessageFromUser>Have you tried ice cream?</MessageFromUser>
<MessageFromBot>I don't know how to respond to that </MessageFromBot>
<MessageAlert>You have reached the end of the conversation</MessageAlert>
```

and so on.

Reply:
IMO table is a good example of a component that does the same
https://www.patternfly.org/components/table#basic

Reply:
Fixed